config.yml

```diff
@@ -1948,6 +1964,11 @@ nodes:
         type: location?
       - name: parts
         type: node[]
+        kind:
+          - StringNode
+          - EmbeddedStatementsNode
+          - EmbeddedVariableNode
+          - SymbolNode # %I example in spanning_heredoc.txt line 48
```

Comment:
This is a bug, will be addressed by https://github.com/ruby/prism/pull/2517, so this can be removed.